apps/www/registry/default/ui/slider.tsx

```diff
@@ -11,14 +11,29 @@ const Slider = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <SliderPrimitive.Root
     ref={ref}
+    // eslint-disable-next-line tailwindcss/no-contradicting-classname
     className={cn(
-      "relative flex w-full touch-none select-none items-center",
+      "relative flex touch-none select-none",
+      "data-[orientation='horizontal']:w-full data-[orientation='horizontal']:items-center",
+      "data-[orientation='vertical']:h-full data-[orientation='vertical']:justify-center",
```

Comment:
for vertical you should add `w-2` to prevent the need for flex, related to https://github.com/shadcn-ui/ui/pull/2586/files#r1564900994

Reply:
Ah I see. Yeah I will fix it tonight! Actually I stated my intent in the PR description, I intentionally didn't add this because I thot it would make sense for parent to be responsible for determining the width of the vertical slider or height of the horizontal slider. But yeah, having a default `w-2` now I think about it does make sense also.